components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -277,6 +286,35 @@ class MozillaSocorroService(
         return map
     }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun readExtrasFromFile(file: File): HashMap<String, String> {
+        var resultMap = HashMap<String, String>()
+        var notJson = false
+
+        try {
+            FileReader(file).use { fileReader ->
+                val input = fileReader.readLines().firstOrNull()
+                input ?: throw JSONException("failed to read json file")
+
+                val jsonObject = JSONObject(input)
+                for (key in jsonObject.keys()) {
+                    resultMap[key] = jsonUnescape(jsonObject.getString(key))
+                }
+            }
+        } catch (e: FileNotFoundException) {
+            Logger.error("failed to find extra file", e)
```

Comment:
nit: We also need to catch `IOException` here.

Reply:
Thanks, I'll add it